### PR TITLE
backend: log username for various controller actions 

### PR DESCRIPTION
**Pull request by vbatts**
this is to log more information during certain events that the users perform

### CODE REVIEW

**Comment by vbatts** [comment]
PTAL

**Review by iaguis — APPROVED** [review]
I updated this a bit:

* Print whole objects when doing changes
* Use structured logging to print the username
* Print old and new state when updating objects
* Don't print TeamID since this is an internal (hardcoded) Nebraska team and not the GitHub team. I think printing the username is enough.

Also, I've tested this and it seems to do what it should.

File: cmd/nebraska/controller.go [hunk 1]
```diff
@@ -481,21 +554,35 @@ func (ctl *controller) addChannel(c *gin.Context) {
...
+	if err != nil {
+		logger.Error().Err(err).Str("channelID", channelID).Msg("updateChannel - getting old channel to update")
+		httpError(c, http.StatusInternalServerError)
+		return
+	}
+
 	channel := &api.Channel{}
 	if err := json.NewDecoder(c.Request.Body).Decode(channel); err != nil {
-		logger.Error().Err(err).Msg("updateChannel - decoding payload")
+		logger.Error().Err(err).Msgf("updateChannel - decoding payload")
```
Comment (joaquimrocha) [inline cmd/nebraska/controller.go:576]: Nitpick: This one is not formatting the message.

File: cmd/nebraska/controller.go [hunk 2]
```diff
@@ -565,9 +667,13 @@ func (ctl *controller) getChannels(c *gin.Context) {
...
 
 func (ctl *controller) addPackage(c *gin.Context) {
+	session := ginsessions.GetSession(c)
+	username := session.Get("username")
+	logger := logger.With().Str("username", username.(string)).Logger()
+
 	pkg := &api.Package{}
 	if err := json.NewDecoder(c.Request.Body).Decode(pkg); err != nil {
-		logger.Error().Err(err).Msg("addPackage - decoding payload")
+		logger.Error().Err(err).Msgf("addPackage - decoding payload")
```
Comment (joaquimrocha) [inline cmd/nebraska/controller.go:676]: Nitpick: Is not formatting the message (there are more like these below).

**Review by joaquimrocha — COMMENTED** [review]
Left a comment (and a couple of nitpicks), but overall LGTM.

**Comment by iaguis** [comment]
Updated

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**